config/initializers/remote_client.rb

```diff
@@ -2,6 +2,6 @@
   client.base_url = Settings.remote_api_url
   client.api_key = Settings.remote_api_key
   client.logger = Rails.logger
-  client.open_timeout = 5
+  client.open_timeout = 10
```

Comment:
Maybe add a comment in the code to indicate why the timeout is set to 10, and that we ideally want it to be lower. You could reference https://dsdmoj.atlassian.net/browse/CFP-587